server/sonar-web/src/main/js/components/common/MultiSelectOption.tsx

```diff
@@ -19,42 +19,53 @@
  */
 import * as React from 'react';
 import * as classNames from 'classnames';
+import { MultiSelectValue } from './MultiSelect';
 
 interface Props {
-  element: string;
+  element: MultiSelectValue;
   selected?: boolean;
+  disabled?: boolean;
   custom?: boolean;
   active?: boolean;
-  onSelectChange: (elem: string, selected: boolean) => void;
-  onHover: (elem: string) => void;
+  onSelectChange: (elem: MultiSelectValue, selected: boolean) => void;
+  onHover: (elem: MultiSelectValue) => void;
 }
 
 export default class MultiSelectOption extends React.PureComponent<Props> {
   handleSelect = (evt: React.SyntheticEvent<HTMLAnchorElement>) => {
+    const { disabled = false } = this.props;
```

Comment:
there is no need to assign default value `false`: `if (disabled) {}` will correctly work with `undefined`

Reply:
and I'd prefer inverting `if`: `if (!this.props.disabled) { this.props.onSelectChange(...) }`